CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+19.0 (2018-04-10)
+-----------------
+
+* Allow details and summary tags (#77)
+* Add .pytest_cache to .gitignore. (#85)
```

Comment:
Remove final period for consistency?

Reply:
Added periods to all of them to be more consistent with the rest of the changelog.